apps/src/p5lab/AnimationTab/NewListItem.jsx

```diff
@@ -15,90 +15,22 @@ class NewListItem extends React.Component {
   };
 
   render() {
-    const hovered = Radium.getState(this.state, 'main', ':hover');
     return (
       <button
-        style={styles.tile}
+        id="newListItem"
+        className={style.tile}
         onClick={this.props.onClick}
-        className="newListItem"
         type="button"
       >
-        <div style={styles.wrapper}>
-          <div style={[styles.border, hovered && styles.borderHovered]}>
-            <i
-              className="fa fa-plus-circle"
-              style={[styles.addButton, hovered && styles.borderHovered_add]}
-            />
-            <div
-              className="animation-name"
-              style={[
-                styles.animationName,
-                hovered && styles.borderHovered_animationName
-              ]}
-            >
-              {this.props.label}
-            </div>
+        <div className={style.wrapper}>
+          <div className={style.border}>
+            <i className={classNames(style.addButton, 'fa fa-plus-circle')} />
+            <div className={style.animationName}>{this.props.label}</div>
```

Comment:
Is the animation-name class name no longer needed?

Reply:
Good call. I couldn't find that className in our repo (it's used as an attribute in some places https://github.com/code-dot-org/code-dot-org/blob/staging/apps/style/common.scss#L47)

It looks like some of the other classNames were used as identifiers in eyes tests, so I'm wondering if that was used at some point here as well.

Reply:
TIL about `animation-name` as a CSS property :)

Reply:
I also didn't know about that until this work!